src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,48 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+    @Test
+    void stringTest1() {
+        String[] multiDigitStrings = "1,2".split(",");
+
+        assertThat(multiDigitStrings).contains("1");
+        assertThat(multiDigitStrings).contains("2");
+
+        assertThat(multiDigitStrings).containsExactly("1", "2");
+
+        String[] singleDigitStrings = "1".split(",");
+
+        assertThat(singleDigitStrings).containsExactly("1");
+    }
+
+    @Test
+    void stringTest2() {
+        String strippedString = "(1,2)".substring(1,4);
+
+        assertThat(strippedString).isEqualTo("1,2");
+    }
+
+    @Test
+    @DisplayName("StringIndexOutOfBoundsExceptionTest")
+    void stringTest3() {
+        String originalString = "abc";
+
+        assertThat(originalString.charAt(1)).isEqualTo('b');
+
+        assertThatExceptionOfType(StringIndexOutOfBoundsException.class)
+                .isThrownBy(() ->{
+                    originalString.charAt(3);
+                })
+                .withMessageMatching("String index out of range: \\d+");
+
+        assertThatThrownBy(() -> {
+            originalString.charAt(3);
+        }).isInstanceOf(StringIndexOutOfBoundsException.class)
```

Comment:
람다 사용시 {} <- 중괄호 표현을 생략하셔도 됩니다 :-) 
```suggestion
        assertThatExceptionOfType(StringIndexOutOfBoundsException.class)
                .isThrownBy(() -> originalString.charAt(3))
                .withMessageMatching("String index out of range: \\d+");

        assertThatThrownBy(() -> originalString.charAt(3)).isInstanceOf(StringIndexOutOfBoundsException.class)
        .hasMessageContaining("String index out of range:");
```